src/managers/KeybindManager.cpp

```diff
@@ -1670,8 +1670,17 @@ void CKeybindManager::focusWindow(std::string regexp) {
         changeworkspace(PWORKSPACE->getConfigName());
     }
 
+    const auto WASFULLSCREEN = g_pCompositor->m_pLastWindow->m_bIsFullscreen;
```

Comment:
m_pLastWindow still can be null. This would be a nullptr dereference aka segfault.

```cpp
g_pCompositor->m_pLastWindow ? g_pCompositor->m_pLastWindow->m_bIsFullscreen : false;
```

Reply:
what about WASFULLSCREEN == true or in case you don't like it, could you suggest a better way?

Reply:
what?

Reply:
nevermind, changed the code, can you check